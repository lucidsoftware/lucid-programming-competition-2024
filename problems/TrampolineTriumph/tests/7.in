9
0.823305 0.830844 0.817229 0.763426 0.777933 0.805124 0.933618 0.932254 0.788894
70
1941.301724 1948.269570 4.219353
2393.162431 2399.174040 8.167576
2253.985812 2263.229510 9.057133
1765.259422 1772.161944 9.782113
242.575069 253.631138 9.393217
1900.310660 1910.648644 7.605159
307.997967 317.796969 6.495689
2254.384527 2260.097400 2.409287
2265.948237 2272.857600 9.008534
1612.842742 1621.622993 9.732324
2128.107620 2138.703967 6.743786
2605.888730 2613.002868 7.492710
1139.071290 1144.187800 9.238282
1525.780056 1533.378570 7.433345
2405.875685 2411.502743 3.882492
1523.629944 1530.059757 4.274379
785.975396 794.356402 3.623689
280.249603 289.992591 6.266858
116.020643 127.681482 2.976461
2078.473222 2084.483308 7.359243
2115.297670 2123.797996 2.537144
230.537267 237.513489 8.159263
2670.177184 2676.484697 7.740630
1286.524338 1293.672090 7.343854
1435.559563 1442.598729 7.253617
2543.403534 2551.074897 6.234588
1099.692418 1106.869708 2.862224
30.668630 36.771773 5.794796
2381.062079 2386.068523 6.184509
45.170650 51.981133 5.446070
2379.158286 2385.256445 7.876442
2793.259863 2800.268989 5.915125
1942.770661 1948.695539 6.691908
2192.978920 2201.616317 7.985288
155.076007 162.345173 2.061440
648.411609 654.072463 5.054605
3336.719539 3342.345674 8.309685
1562.543528 1568.255304 9.972177
557.610491 566.505469 3.506723
1160.646101 1167.423441 7.330452
2343.150612 2349.017531 4.839575
1386.947582 1392.965872 3.816890
2894.992458 2901.800998 5.288373
1503.136096 1508.186385 4.756096
2969.943945 2977.318380 8.967004
1765.827354 1770.837546 7.773625
2302.638625 2309.799823 4.152697
973.072493 979.951059 6.099485
2090.068023 2098.175370 7.792417
608.882223 618.678582 3.965388
1977.986159 1983.693904 5.154864
661.216930 667.306510 2.539051
2017.153587 2022.157429 6.279975
2303.384219 2312.282555 6.310137
1929.057446 1935.535762 7.794909
2468.516089 2473.815055 7.659589
535.145173 540.423186 3.175610
2140.462496 2150.321776 5.136477
2670.230479 2678.684966 6.875324
1526.465353 1536.572959 2.680621
30.348690 39.803993 8.661179
2029.874333 2037.782722 8.195825
645.218803 651.976720 5.003372
1877.841958 1886.995945 9.563868
270.134540 275.340468 3.586502
2969.134888 2980.117769 5.272544
648.836789 657.337073 5.774471
406.780594 416.142182 7.716346
1200.998336 1206.051975 5.406816
2006.449328 2013.201681 8.478760
