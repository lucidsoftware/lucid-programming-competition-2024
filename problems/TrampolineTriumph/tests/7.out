8
423.515162
7
414.765402
6
384.027050
1
370.481739
3
367.340074
2
366.634907
5
361.127022
4
360.444070
9
352.350519
